root/meta/rootmap/duplicateWarning/CMakeLists.txt

```diff
@@ -2,5 +2,5 @@ configure_file(rfile1.rootmap . COPYONLY)
 configure_file(rfile2.rootmap . COPYONLY)
 
 ROOTTEST_ADD_TEST(myclassduplicated
-                  COMMAND root -b -q -l
+                  COMMAND ${ROOT_root_CMD} -b -q -l
                   PASSREGEX  "Warning in <TInterpreter::ReadRootmapFile>: While processing ./rfile[12]\.rootmap, class myclassduplicated was found to be associated to libdup[12]\.so although it is already associated to libdup[12]\.so")
```

Comment:
Not sure about the `.so` suffix on Windows. It will most probably complain about `.dll`

Reply:
Well, in fact it doesn't

Reply:
I think it's because `root/meta/rootmap/duplicateWarning/rfile1.rootmap` and `rfile2.rootmap` literally specify `libdup1.so` and `libdup2.so`, respectively.

Reply:
Yes, I saw it (a bit too late unfortunately, hence the comment and then solving the conversation. Sorry for the trouble